src/table/Source.cc

```diff
@@ -175,12 +175,13 @@ void SourceFitsWriter::_writeRecord(BaseRecord const & r) {
 //----- SourceFitsReader ------------------------------------------------------------------------------------
 //-----------------------------------------------------------------------------------------------------------
 
-// A custom FitsReader for Sources - this reads footprints as variable-length arrays, and adds header
-// keys that define the slots.  It gets registered with name SOURCE, so it should get used whenever
-// we read a table with AFW_TYPE set to that value.
+// A custom FitsReader for Sources - this reads footprints in addition to the regular fields.  It
```

Comment:
Instead I've mentioned in that comment that the reader class itself is after some helper classes.  I'd like to keep the description up here so someone reading down from the top of the file understands the context for the helper classes.